### ISSUE TITLE
blacklist 'flow' doesn't keep annotations on destructured arguments

### ISSUE THREAD

**Issue by ccapndave** [babel/babel#1461]
Specifically:

```js
/*::
type MyFuncArgs = { something: any, idx: number };
*/
function myFunc({ something, idx }: MyFuncArgs) { }
```

is being transpiled into:

```js
/*::
type MyFuncArgs = { something: any, idx: number };
*/
function myFunc(_ref) {
	var something = _ref.something;
	var idx = _ref.idx;
}
```

This causes Flow to complain about: `parameter _ref - Missing annotation`

**Comment by ccapndave** [comment]
4 minutes turnaround?  That was incredible :)

**Comment by sebmck** [comment]
Fixed as of 5.2.16, thanks!
